core/src/main/java/io/grpc/internal/AbstractServerStream.java

```diff
@@ -228,17 +230,8 @@ public final void abortStream(Status status, boolean notifyClient) {
     // TODO(louiscryan): Investigate whether we can remove the notification to the client
     // and rely on a transport layer stream reset instead.
     Preconditions.checkArgument(!status.isOk(), "status must not be OK");
+    inboundPhase(Phase.STATUS);
     closeListener(status);
-    if (notifyClient) {
```

Comment:
Actually, the only places it's called now are in NettyServerHandler:

onChannelInactive: we can't send RST_STREAM, since the channel is broken.

onRstStreamRead: No point in sending RST_STREAM

onStreamError: Modified so that it will send RST_STREAM after calling abortStream.